### DISCUSSION TITLE
QR factorization with column pivoting

### DISCUSSION THREAD

**Discussion by wdecre** [giaf/blasfeo#146] [Ideas]
In one of our projects we want to perform a QR factorization with column pivoting.

Are there plans to include this in BLASFEO? Or is maybe someone already working on this? If there are no plans, but if it would be of interest, we are of course open to contribute.

Wilm

**Comment by giaf** [comment]
First of all, sorry for the late response and thanks for the interest in BLASFEO.

No there are currently no plans to add QR factorization with column pivoting to BLASFEO.
It could be of interest, but the workload needed for that depends quite a lot of what your target is, in terms of:
- target architecture
- panel- vs column-major matrix format
- double or single precision

When you say you could contribute, can you explain in which sense would you offer to contribute?
